tools/docker-images/clp-execution-base-focal/setup-scripts/install-prebuilt-packages.sh

```diff
@@ -5,4 +5,5 @@ DEBIAN_FRONTEND=noninteractive apt-get install -y \
   checkinstall \
   curl \
   python3 \
-  rsync
+  rsync \
+  zstd
```

Comment:
Can we instead install from source so that it matches up with the version linked with clp? The ones from package managers may be behind the version we use with CLP since we try to take advantage of the latest fixes and improvements.

Reply:
Based on our out-of-band discussion, installing the source package has a few challenges:

* The execution image doesn't have the necessary build tools
* It requires some more work to keep the version installed in this image in-sync with the version used by clp-core.

Thus, we've decided to continue using zstd from the package manager.